configure.ac

```diff
@@ -301,6 +301,7 @@ AC_CHECK_FUNCS( futimes )
 AC_CHECK_FUNCS( wcslcpy lrand48_r killpg )
 AC_CHECK_FUNCS( backtrace_symbols getifaddrs )
 AC_CHECK_FUNCS( futimens clock_gettime )
+AC_CHECK_FUNCS( localtime_r )
```

Comment:
What is the point of this configure check? This change calls `localtime()` unconditionally. Which is reasonable since no OS fish runs on should not have an implementation for that function.